lib/registries/addon/components/comma-list/template.hbs

```diff
@@ -0,0 +1,15 @@
+<ul local-class="CommaList__List">
+    {{~#each filteredItems as |item index|~}}
+        <li local-class="CommaList__Item {{if (gt index cutOff) 'CommaList__Item--Hidden' ''}} {{if (eq index cutOffOrLast) 'CommaList__Item--Last' ''}}">
```

Comment:
This feels like maybe too much logic in the template. How about having a `comma-list/x-item` subcomponent and using `@localClass` to bind local classes?